invocation/invocation_trace_support.go

```diff
@@ -1,13 +1,136 @@
 package invocation
 
 import (
+	"errors"
+	"strings"
+
 	"github.com/thundra-io/thundra-lambda-agent-go/config"
 	"github.com/thundra-io/thundra-lambda-agent-go/constants"
 	"github.com/thundra-io/thundra-lambda-agent-go/trace"
+	"github.com/thundra-io/thundra-lambda-agent-go/tracer"
 )
 
 var incomingTraceLinks = make([]string, 0)
 
+// Resource type stores information about resources in spans
+type Resource struct {
+	ResourceType       string   `json:"resourceType"`
+	ResourceName       string   `json:"resourceName"`
+	ResourceOperation  string   `json:"resourceOperation"`
+	ResourceCount      int      `json:"resourceCount"`
+	ResourceErrorCount int      `json:"resourceErrorCount"`
+	ResourceDuration   int64    `json:"resourceDuration"`
+	ResourceErrors     []string `json:"resourceErrors"`
+	resourceErrorsMap  map[string]struct{}
```

Comment:
Is `resourceErrorsMap ` ignored while serializing resource into JSON?

Reply:
Yes